controller/server/server.go

```diff
@@ -35,7 +35,12 @@ type Server struct {
 
 // CreateServer creates an instance of the service controller server.
 func CreateServer(w *watch.Watcher) (*Server, error) {
-	c := createController(storage.CreateTPRStorage(w))
+	c, err := createController(storage.CreateTPRStorage(w))
+	if err != nil {
+		log.Printf("Couldn't create controller: %v\n", err)
```

Comment:
think we should use glog throughout the codebase. It's introduced in https://github.com/kubernetes-incubator/service-catalog/pull/153, so it seems like something to come back to in a follow-up